delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/acls/AclAddress.scala

```diff
@@ -29,6 +29,9 @@ object AclAddress {
   final private[sdk] val orgAddressRegex  = s"^/(${Label.regex.regex})$$".r
   final private[sdk] val projAddressRegex = s"^/(${Label.regex.regex})/(${Label.regex.regex})$$".r
 
+  implicit final def fromProject(project: ProjectRef): Project = Project(project)
```

Comment:
I am not so fond of this implicit magic

Reply:
In this case it makes total sense.

Reply:
it does make things simpler at the call site...